frontend/src/pages/logout/logout.loader.ts

```diff
@@ -9,5 +9,11 @@ export type LogoutLoaderData = void;
  */
 export async function logoutLoader(): Promise<LogoutLoaderData> {
   await cacheDelete("", true); // Clears all cache.
-  await logout();
+  sessionStorage.clear();
```

Comment:
I think we can use maykinmedia/client-common's cacheDelete call with:

- key = ""
- startsWith = true

This should clear only the namespaced cache records which might be a little cleaner. But since sessionStorage is domain scoped and possible conflicts should be limited to localhost. This is ok for now.